source/common/upstream/cluster_manager_impl.h

```diff
@@ -930,6 +931,8 @@ class ClusterManagerImpl : public ClusterManager,
   bool initialized_{};
   bool ads_mux_initialized_{};
   std::atomic<bool> shutdown_{};
+
+  std::size_t last_recorded_warming_clusters_count_{0};
```

Comment:
comments please. bonus points for why the stat "of the same value" isn't safe

Reply:
Done